sktime/transformers/shapelets.py

```diff
@@ -313,32 +313,32 @@ def _round_robin(*iterables):
                         shapelet_heaps_by_class[this_class_val].pop()
 
                 # Takes into account the use of the MAX shapelet calculation time to not exceed the time_limit (not exact, but likely a good guess).
-                if hasattr(self,'time_limit') and self.time_limit > 0:
+                if hasattr(self,'time_limit_in_mins') and self.time_limit_in_mins > 0:
                     time_now = time_taken()
                     time_this_shapelet = (time_now - time_last_shapelet)
                     if time_this_shapelet > max_time_calc_shapelet:
                         max_time_calc_shapelet = time_this_shapelet
                     time_last_shapelet = time_now
-                    if (time_now + max_time_calc_shapelet) > self.time_limit:
+                    if (time_now + max_time_calc_shapelet) > self.time_limit_in_mins * 60:
                         if self.verbose > 0:
                             print("No more time available! It's been {0:02d}:{1:02}".format(int(round(time_now / 60, 3)), int((round(time_now / 60, 3) - int(round(time_now / 60, 3))) * 60)))
                         time_finished = True
                         break
                     else:
                         if self.verbose > 0:
                             if candidate_rejected is False:
-                                print("Candidate finished. {0:02d}:{1:02} remaining".format(int(round((self.time_limit - time_now) / 60, 3)),
-                                                                                            int((round((self.time_limit - time_now) / 60, 3) - int(round((self.time_limit - time_now) / 60, 3))) * 60)))
+                                print("Candidate finished. {0:02d}:{1:02} remaining".format(int(round(self.time_limit_in_mins - time_now / 60, 3)),
+                                                                                            int((round(self.time_limit_in_mins - time_now / 60, 3) - int(round(self.time_limit_in_mins - time_now / 60, 3))) * 60)))
                             else:
-                                print("Candidate rejected. {0:02d}:{1:02} remaining".format(int(round((self.time_limit - time_now) / 60, 3)),
```

Comment:
Why do we remove the parenthesis around the subtraction here? Does that not change the calculation?

Reply:
`time_now` is in seconds, and `self.time_limit` used to be equal to `time_limit_in_mins * 60`, i.e. also seconds. This multiplication by 60 in the constructor was removed to stay consistent with the naming and sklearn conventions. `self.time_limit_in_mins` is therefore in minutes and not in seconds. 

In order to keep `(self.time_limit - time_now) / 60 = (self.time_limit_in_minutes * 60 - time_now) /60` the same, we simply remove the parantheses and get `self.time_limit_in_minutes - time_now/60`.